src/logic/calculate.js

```diff
@@ -0,0 +1,97 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
```

Comment:
* [ ] let us maintain the default value of total to '0' as stated in the earlier milestone, hence when 'AC' is selected total should return to "0". that way user would read "0" from their screen instead of null.